patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -520,6 +531,11 @@
 +    }
 +
 +    private String displayname;
++    
++    /** A collection of components to prepend to the username. This is not persisted across restarts */
```

Comment:
I would personally split this off into ForgeHooks or a helper class and have addPrefix and addSuffix setter methods, instead of making them public, which has potential for abuse.